src/main/cpp/auton/AutonSelector.h

```diff
@@ -63,9 +72,15 @@ class AutonSelector
 	void PutChoicesOnDashboard();
 	bool FileExists(const std::string &name);
 	bool FileValid(const std::string &name);
+
+	static AutonSelector *m_instance;
 	// Attributues
 	frc::SendableChooser<std::string> m_startposchooser;
 	frc::SendableChooser<std::string> m_targetFace;
 	frc::SendableChooser<std::string> m_targetGamePiece;
 	frc::SendableChooser<std::string> m_desiredScoringLevel;
+
+	const std::string m_startDelay = "Start Delay";
```

Comment:
Wonder if the issue you were seeing was due to the spaces in the string; might try removing and testing again